module-2/backend/week_four.md

```diff
@@ -10,14 +10,48 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 * What is a cookie?
+A cookie is a file that is attached to the response you send a user and it gets stored on their browser so that the next time they come back to your site, the cookie gets sent back to you. It is a way of tricking the computer into remembering informatin about the user.
```

Comment:
🎉 another explanation might be: a small piece of data sent from a website and stored on the user's computer by the user's web browser while the user is browsing.